tools/Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = ["**/src/.."]
 resolver = "2"
 
 [workspace.dependencies]
-anyhow = "1.0.75"
+anyhow = "1.0.80"
```

Comment:
It should be .... but its message is that it sticks to a particular version. I do not buy the argument of locking to an older version, after all, we regularly update versions. Then, if we agree that `=1.0.80` is semantically equivalent to `=1.0` then the latter is shorter and prettier.

Reply:
I meant someone along the chain. We can't ensure a dependency doesn't drag **us** down because **it** locked down. And, no, `=X.Y.Z` was never semantically equivalent `=X.Y`.

If you want really want to use versions as short as possible, okay, I'll follow that. I just want to make sure you are aware of the risks.

Reply:
As an example of a similar situation, I had to turn off the dependency auditing for quite a while because a dependency locked it's dependency to a specific version. Mind you, that locked dependency was `webpki` and had a high-risk CVE.

Things like this do happen, and people do lock their dependencies sometimes.

Reply:
I do not insist, just wanna follow the same convention throughout the repo